mmf/datasets/processors/processors.py

```diff
@@ -1385,4 +1385,28 @@ def __call__(self, item):
             else:
                 # no masking token (will be ignored by loss function later)
                 image_labels.append(-1)
-        return image_labels
+        return torch.tensor(image_labels, dtype=torch.long)
+
+
+@registry.register_processor("transformer_bbox")
+class TransformerBboxProcessor(BaseProcessor):
+    def __init__(self, config, *args, **kwargs):
+        super().__init__(config, *args, **kwargs)
+
+    def __call__(self, item):
+        bbox = item["bbox"]
```

Comment:
This can be made parameters for processor config. Including `bbox_key`, `image_width` and `image_height`.